bake/hcl_test.go

```diff
@@ -674,7 +674,7 @@ func TestHCLAttrsCapsuleTypeVars(t *testing.T) {
 		output = [ "type=oci,dest=../${foo}.tar" ]
 
 		secret = [
-			{ id = target.app.output[0].type, src = "/local/secret" },
+			{ id = target.app.output[0].type, src = "/${target.app.cache-from[1].type}/secret" },
```

Comment:
@dvdksn I'm not sure we have examples showing how to reference attributes. I think the example with `cache-to = [ target.app.cache-from[0] ]` is a good one.

Reply:
We have a simpler example: https://docs.docker.com/build/bake/inheritance/#reusing-single-attributes-from-targets

We don't show interpolation though, maybe we should